locale/en-us/wifi_intro_3.dialog

```diff
@@ -1 +1 @@
-Open your browser and go to start dot openvoiceos dot com and choose the Wifi network from the list you would like to connect your OVOS device with
+Open your browser and go to {{url}} and choose the Wifi network from the list you would like to connect your device to
```

Comment:
This always struck me as droning. Could it be shorter?

"Open your browser and go to {url} to continue setup" seems like enough information. If it needs to explain the page verbally, it can react to a page hit by speaking, but I think the menu is pretty much the same as every other wifi menu users will have used.

Reply:
I have no objections to this; will make that minor dialog change